src/lib/routing-table.js

```diff
@@ -102,4 +124,12 @@ class RoutingTable {
   }
 }
 
+function getPrefixLength (str1, str2) {
```

Comment:
The former is how route prefixes are supposed to work though, isn't it? I thought that the `.` separator was just convention.